<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>海贼旗子</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/OnePiece.png</normaloff>:/Image/OnePiece.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>191</width>
     <height>164</height>
    </rect>
   </property>
   <property name="title">
    <string>问卷调查</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="cBox1">
      <property name="text">
       <string>环境优美</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cBox2">
      <property name="text">
       <string>服务良好</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cBox3">
      <property name="text">
       <string>饭菜好吃</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>177</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>性别</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rBtnMan">
         <property name="text">
          <string>男</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnWoman">
         <property name="text">
          <string>女</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>婚否</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rBtnMarried">
         <property name="text">
          <string>已婚</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnUnmarried">
         <property name="text">
          <string>未婚</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
